### ISSUE TITLE
useless code on HTTPClient.start()

### ISSUE THREAD

**Issue by umisama** [masawada/violet#4]
http.js - line : 24-26

``` javascript
proto.start = function(){
  var xhr = this.request, self = this;
  xhr.open(this.method, this.uri, true);
```

defining self is useless.

**Comment by umisama** [comment]
fixed with https://github.com/masawada/violet/pull/5

**Comment by masawada** [comment]
Merged 600a2e0106386622c5b4399b056dcd4b742516ad
